examples/audio_analysis_for_track.py

```diff
@@ -10,10 +10,7 @@
 client_credentials_manager = SpotifyClientCredentials()
 sp = spotipy.Spotify(client_credentials_manager=client_credentials_manager)
 
-if len(sys.argv) > 1:
-    tid = sys.argv[1]
-else:
-    tid = 'spotify:track:4TTV7EcfroSLWzXRY6gLv6'
+tid = sys.argv[1] if len(sys.argv) > 1 else 'spotify:track:4TTV7EcfroSLWzXRY6gLv6'
```

Comment:
I won't add these changes, it makes the code a bit harder to read, considering these are just examples.